gtfs_realtime_translators/translators/septa.py

```diff
@@ -3,21 +3,19 @@
 
 import pendulum
 
-from gtfs_realtime_translators.factories import TripUpdate, FeedMessage
+from gtfs_realtime_translators.factories import TripUpdate, FeedMessage, RequiredFieldValidator
 
 
-class VtaGtfsRealtimeTranslator:
-    def __init__(self, stop_id=None):
-        if stop_id is None:
-            raise ValueError('stop_id is required.')
-
-        self.stop_id = stop_id
+class SwiftlyGtfsRealtimeTranslator:
```

Comment:
The filename should be renamed `swiftly.py`, not something org specific

Reply:
Make sense. will do.